### PR TITLE
fix: intermittent failures in `feature_governance.py`

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
Should fix failures like https://gitlab.com/dashpay/dash/-/jobs/6107697452

## What was done?
See inline comments

## How Has This Been Tested?
Run lots of `feature_governance.py` in parallel multiple times

## Breaking Changes
n/a

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by ogabrielides — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

File: test/functional/feature_governance.py [hunk 1]
```diff
@@ -77,6 +77,19 @@ def check_superblock(self):
...
 
+    def have_trigger_for_height(self, sb_block_height):
+        count = 0
+        for node in self.nodes:
+            valid_triggers = node.gobject("list", "valid", "triggers")
+            valid_triggers_list = list(valid_triggers.values())
+            for i in range(len(valid_triggers)):
+                if json.loads(valid_triggers_list[i]["DataString"])["event_block_height"] != sb_block_height:
+                    continue
+                if valid_triggers_list[i]['AbsoluteYesCount'] > 0:
```
Comment (knst) [inline test/functional/feature_governance.py:88]: nit: why not something like?

```suggestion
            for trigger in list(valid_triggers.values()):
                if json.loads(trigger["DataString"])["event_block_height"] != sb_block_height:
                    continue
                if trigger['AbsoluteYesCount'] > 0:
```

(I haven

**Review by knst — COMMENTED** [review]
I can't reproduce issue locally, but LGTM except a mentioned nit

**Review by knst — APPROVED** [review]
LGTM

**Review by ogabrielides — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**